From cf969d037114abd2519f77bf2a4e7e13c32fe7c2 Mon Sep 17 00:00:00 2001
From: Franz-Josef Haider <franz.haider@jolla.com>
Date: Wed, 6 Jun 2018 09:30:01 +0200
Subject: [PATCH 27/37] (hybris) load services from droid-hybris as early as
 possible

This makes sure our service definitions are loaded first and can be used to
overwrite service definitions which are slightly different to what we need and
cannot be fixed otherwise.

Change-Id: I58e86f8b3c65f46c24fc3102a859867bccb6a713
---
 init/init.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/init/init.cpp b/init/init.cpp
index cdab577..1ffa036 100644
--- a/init/init.cpp
+++ b/init/init.cpp
@@ -116,6 +116,9 @@ static void LoadBootScripts(ActionManager& action_manager, ServiceList& service_
         if (!parser.ParseConfig("/usr/libexec/droid-hybris/system/etc/init")) {
             late_import_paths.emplace_back("/usr/libexec/droid-hybris/system/etc/init");
         }
+        if (!parser.ParseConfig("/system/etc/init")) {
+            late_import_paths.emplace_back("/system/etc/init");
+        }
         if (!parser.ParseConfig("/product/etc/init")) {
             late_import_paths.emplace_back("/product/etc/init");
         }
-- 
2.7.4

